kube/resources/resource-defaults.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: LimitRange
```

Comment:
I don't think this will work as intended if someone deploys Airbyte into the same namespace as another deployment.

Reply:
Ok, so I do have to define it for each container separately...

Similar to `namespace` it doesn't seem possible to define the values from an environment variable though

Reply:
that matches what I've seen around online